Tests_v3/Virtual Time/VirtualTimer.swift

```diff
@@ -27,6 +27,10 @@ struct VirtualTimeTimer: StreamChatClient.Timer {
             callback: { _ in onFire() }
         )
     }
+    
+    static func currentTime() -> Date {
+        Date(timeIntervalSinceReferenceDate: time.currentTime)
+    }
```

Comment:
I don't understand this. Here you combine the actual time with the virtual time? 🤯